server/src/models/Color.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose';
```

Comment:
I guess you are right, and we may not really need a separate entity for colours. We'll store standard colours in the code, and custom user colours can be stored directly in the User collection.